examples/tutorials/vectors.py

```diff
@@ -0,0 +1,177 @@
+"""
+Plot vectors
+==========
+
+Plotting vectors is handled by :meth:`pygmt.Figure.plot`.
+
+.. note::
+
+    This tutorial assumes the use of a Python notebook, such as IPython or Jupyter Notebook.
+    To see the figures while using a Python script instead, use
+    ``fig.show(method="external")`` to display the figure in the default PDF viewer.
+
+    To save the figure, use ``fig.savefig("figname.pdf")`` where ``"figname.pdf"``
+    is the desired name and file extension for the saved figure.
+"""
+# TODO: change this number to reflect the correct thumbnail
+# sphinx_gallery_thumbnail_number = 3
+
+import numpy as np
+import pygmt
+
+#######################################################################################
+# Plot vectors
+# ----------
+#
+# Create a Cartesian figure using ``projection`` parameter and set the axis scales
+# using ``region`` (in this case, each axis is 0-25). Pass a ``numpy`` array object
+# that contains lists of all vectors to be plotted.
+
+# Vector specifications are structured as: [x_start, y_start, direction_degrees, magnitude]
```

Comment:
Thanks for this clarification. I did not know that these are the defaults. So the user can change the units of the projection by changing `X5c/5c` to be something like `X5i/5i` for example. But is it possible for the user to change the units for the vector magnitude? Since the list of the vector information only has ints or floats, where would that parameter be passed?
Thanks again for all the help!

Reply:
> But is it possible for the user to change the units for the vector magnitude? Since the list of the vector information only has ints or floats, where would that parameter be passed?

Yes, the units for vector magnitude are controlled by [PROJ_LENGTH_UNIT](https://docs.generic-mapping-tools.org/latest/gmt.conf.html#term-PROJ_LENGTH_UNIT) which is **c** (cm) by default. So when we use **c** for all the parameters, it is okay to forget the units (e.g., your examples). But when **i** (inch) is used in the **projection** parameter, we may also want to use **i** as the unit of vector magnitude. We can change the unit locally or globally using the [pygmt.config](https://www.pygmt.org/latest/api/generated/pygmt.config.html#pygmt.config) method.

Below is two examples with the units of projection parameters are **c** (ex 1) and **i** (ex 2). In each example, three cases are shown: (1) no unit is indicated so the default (**c**) is used; (2) change the unit to be **c** locally; (3) change the unit to be **i** locally. Other parameters are the same, but I repeat them explicitly so that everything is clear.

```Python
import pygmt
data1 = [[2, 3, 0, 2]]
data2 = [[2, 4, 0, 2]]
data3 = [[2, 5, 0, 2]]
```

**EX 1**:
```Python
fig = pygmt.Figure()
fig.plot(region=[0, 10, 0, 10], projection="X10c/10c", frame="af", data=data1, style="v0.6c+e", pen="2p", color="red3")
with pygmt.config(PROJ_LENGTH_UNIT="c"):
    fig.plot(region=[0, 10, 0, 10], projection="X10c/10c", frame="af", data=data2, style="v0.6c+e", pen="2p", color="red3")
with pygmt.config(PROJ_LENGTH_UNIT="i"):
    fig.plot(region=[0, 10, 0, 10], projection="X10c/10c", frame="af", data=data3, style="v0.6c+e", pen="2p", color="red3")
fig.show()
```

**EX 2**:
```Python
fig = pygmt.Figure()
fig.plot(region=[0, 10, 0, 10], projection="X10i/10i", frame="a", data=data1, style="v0.6c+e", pen="2p", color="red3")
with pygmt.config(PROJ_LENGTH_UNIT="c"):
    fig.plot(region=[0, 10, 0, 10], projection="X10i/10i", frame="a", data=data2, style="v0.6c+e", pen="2p", color="red3")
with pygmt.config(PROJ_LENGTH_UNIT="i"):
    fig.plot(region=[0, 10, 0, 10], projection="X10i/10i", frame="a", data=data3, style="v0.6c+e", pen="2p", color="red3")
fig.show()
```

EX 1  |  EX 2
--- | ---
<img src="https://user-images.githubusercontent.com/50591376/111901667-75ac5600-8a74-11eb-84ed-3092f6c97206.png" width=400 /> | <img src="https://user-images.githubusercontent.com/50591376/111901669-76dd8300-8a74-11eb-9a3c-62d8059c1cc6.png" width=400 />

---

I think you don't have to explain the unit in detail. I made a simple revision in the comment above and you could revise it.

Reply:
That makes much more sense now! Thank you again. We will definitely include an example of that and explain what units are being used

Reply:
We just updated the main branch of our fork with the changes you mentioned. Can you please check them out and let us know if the explanation on how to change units is detailed enough. Thanks!

Reply:
Looks great. But the y-axis is a little long, maybe a 10x4 is better than 10x10.

Reply:
Sounds good! I'll change to 10X4 inches